db/schema.rb

```diff
@@ -39,6 +39,18 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "votes", force: :cascade do |t|
+    t.integer "user_id", null: false
+    t.string "votable_type", null: false
+    t.integer "votable_id", null: false
+    t.integer "value"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["user_id"], name: "index_votes_on_user_id"
+    t.index ["votable_type", "votable_id"], name: "index_votes_on_votable"
+  end
+
   add_foreign_key "comments", "articles"
   add_foreign_key "comments", "users"
+  add_foreign_key "votes", "users"
 end
```

Comment:
It doesn't look like your new index got added to the schema. If you are going to change an already existing migration, you should roll it back before you make the changes, then migrate it again afterwards (or just instead of editing the existing migration, you can just create a new one for the extra work you are doing).